models/tealium-event.js

```diff
@@ -161,7 +161,7 @@ class TealiumEvent {
         }
       })
     }
-
+    console.log(this.event.data[TealiumEvent.TAXONOMY_CONTEXT])
```

Comment:
Will this error if `TealiumEvent.TAXONOMY_CONTEXT` is an invalid index?

Reply:
I believe it will just log ```undefined``` if that index doesn't exist.

Reply:
I'll go ahead and add a check though just to be safe. If it doesn't log anything than we know that it's incorrect.